.gitlab-ci.yml

```diff
@@ -2,6 +2,7 @@ image: cuda10.0-mpi-hdf5-mpl
 
 # Is performed before the scripts in the stages step
 before_script:
+    - apt-get install python3.7
```

Comment:
Why is this needed?

Reply:
CI was failing because default has python2.7.
Maybe a red herring.

Reply:
@JoshuaSBrown I'm still debugging the CI.  Will update when done.

Reply:
This has been removed.  CI issues were resolved without this additional install.